src/components/education/EducationForm.jsx

```diff
@@ -0,0 +1,110 @@
+import { useState } from "react"
+
+const EducationForm = () => {
+    const [formData, setFormData] = useState ({
+        course: '',
+        school: '',
+        start_date: '',
+        end_date: '',
+        grade: '',
+        logo: ''
+    })
+
+    const handleChange = (field) => (e) => {
+        e.preventDefault();
+
+        setFormData({
+            ...formData,
+            [field]: e.currentTarget.value
+        })
+    }
+
+    const handleSubmit = (e) => {
+        e.preventDefault();
+
+        fetch('/resume/education', {
+            method: 'POST',
+            headers: {
+                'Content-Type': 'application/json',
+            },
+            body: JSON.stringify(formData)
+        })
+        .then(response => response.json())
+        .then(data => {
+            alert("Education successfully added!")
+        })
+        .catch((error) => {
+            alert("Error: Education not added :(")
```

Comment:
Nice use of alertbox, we can also console log the actual error

Reply:
Added console log of the error